mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraIndexTableHandler.java

```diff
@@ -129,8 +129,7 @@ public Mono<Void> updateIndexOnAdd(MailboxMessage message, CassandraId mailboxId
             .then();
     }
 
-    public Mono<Void> updateIndexOnAdd(Collection<MailboxMessage> messages, CassandraId mailboxId) {
-        int lowConcurrency = 2;
+    public Mono<Void> updateIndexOnAdd(Collection<MailboxMessage> messages, CassandraId mailboxId, int reactorConcurrency) {
```

Comment:
Please use `fairConcurrency` variable like below and do not leak this detail outside.

'Index' projections are not doing LWT thus enabling concurrency here will not result in CAS contension,